package.json

```diff
@@ -38,12 +38,16 @@
     "imagely": "imagely.bin.js"
   },
   "dependencies": {
-    "bluebird": "^2.10.2",
-    "image-size": "^0.3.5",
-    "lodash": "^3.10.1",
-    "phantom": "^0.8.0",
-    "phantomjs": "^1.9.18",
-    "request-promise": "^0.4.3",
-    "yargs": "^3.26.0"
+    "bluebird": "^3.5.0",
+    "image-size": "^0.5.1",
+    "imagely": "imagely-v0.1.12.tgz",
+    "lodash": "^4.17.4",
+    "phantom": "1.x",
+    "phantomjs": "^2.1.7",
+    "request-promise": "^4.1.1",
+    "yargs": "^7.0.1"
+  },
+  "devDependencies": {
+    "yarn": "^0.21.3"
```

Comment:
My thinking was because you need it to develop, but not to run the application. I'll defer to whatever you think is best.